http-transport-binding.md

```diff
@@ -162,12 +162,17 @@ efficient transfer and without transcoding effort.
 For the _binary_ mode, the HTTP `Content-Type` value maps directly to the
 CloudEvents `datacontenttype` attribute.
 
-#### 3.1.2. Event Data Encoding
+#### 3.1.2. HTTP Content-Encoding
+
+For the _binary_ mode, the HTTP `Content-Encoding` value maps directly to the
```

Comment:
Don't understand `maps directly`. Is it:
a) Should be identical to
b) Supersedes
c) If one is present, the other is not needed

Reply:
This language is copied from 3.1.1.

I believe that it is trying to say "If you have a `datacontentencoding` attribute in the CloudEvent, set the `Content-Encoding` header in HTTP. If you have a `Content-Encoding` header in HTTP, set the `datacontentencoding` attribute in the CloudEvent."

This is about mapping an abstract CloudEvent to a specific transport in a bidirectional manner. This means that you either:

1. Have a CloudEvent and want an HTTP rendering
2. Have an HTTP message and want to derive a CloudEvent

In your ontology, I believe this is closest to (a), but (b) and (c) don't make sense to me as options, because you can't have both at once.

Reply:
Fixed the exception on line 178 to include `datacontentencoding`, thanks for helping me notice.

Reply:
Your explanation is so much better than the original obtuse "maps to". Suggest adopting it!

Reply:
Any other opinions on this?

I found "maps to" fairly intuitive and much shorter, but if this is a cause for confusion, I'm happy to spell it out further.

Reply:
I'm ok either way... but rather than having to spell it out for every attribute, perhaps something like: ... the HTTP xxx header's value is populated from the CloudEvents `xxx` attribute.

Although, that sure sounds like "mapping"  :-)

Reply:
Sold, I'll update this.

Reply:
Ugh, it actually is a bidirectional mapping. I'll try "corresponds to".

Reply:
Updated.